Automattic/sensei
# Style the "Awaiting Grade" notice

Resolves https://github.com/Automattic/sensei/issues/7074.

## Proposed Changes
- Updates the styles for the "Awaiting Grade" notice on the quiz page.
- Although the scope was only for the notice on the quiz page, I also updated just the colors of the "Awaiting Grade" notice on the lesson page. Since I broke it with my changes, the easiest fix was to match the colors. Matching the fonts would also be nice for consistency's sake, but that is outside the scope of this PR.
- Unfortunately, the HTML of the _Pending Teacher Grade_ and the _Restart Quiz_ link in the notice and in the quiz footer are different. We probably should have synced up better on this one. Perhaps one day soon we'll be able to match the markup and use common styles. 🙏🏻 
- Made a change to ensure the third-party theme's `learning-mode.css` file loads after the generic Sensei one. Now we won't have to make theme CSS selectors more specific than the default selectors in order to override them. Instead, we just need to match them, and the styles loaded last (i.e. the theme's styles) win.

## Testing Instructions


1. Activate Divi or Astra.
2. Add a quiz to a lesson and ensure that quiz is *not* set to auto-grade.
3. Take the course as a student.
4. View the "Awaiting grade" notice on the quiz page and ensure it matches the design.
5. Verify that the colors of the "Awaiting grade" notice on the lesson page matches that of the quiz page (it's expected that the fonts won't match).
6. Test on mobile as well.
7. Check out this branch - https://github.com/Automattic/themes/pull/7401.
8. Activate the Course theme.
9. Repeat steps 4 & 5 for all style variations.
10. Disable the Quiz Graded email and view the notice again.
11. Verify that "You’ll receive an email once it’s ready to view." is no longer part of the message.
12. Bonus: Disable the `email_customization` feature flag and repeat step 11.

## Pre-Merge Checklist
